test/verify/check-testlib

```diff
@@ -278,6 +278,7 @@ class TestTestlib(testlib.MachineCase):
         self.assertNotIn("cockpittest", self.machine.execute("cat /etc/passwd"))
         self.machine.execute("test ! -e /home/cockpittest")
 
+    @testlib.skipWsContainer("no cockpit-system package installed")
```

Comment:
But I do see cockpit-system as one of the packages being installed in `test/ws-container.install`.

Reply:
But this did work on CoreOS ofcourse

Reply:
ah right -- because we test that one not in beiboot mode, but with fully privileged ws integration. So we need a more specialized variant of that like skipWsBeiboot. This applies to a couple of other tests.